src/eftest/runner.clj

```diff
@@ -84,10 +84,14 @@
   ([vars opts]
    (let [start-time (System/currentTimeMillis)]
      (if (empty? vars)
-       (println "No tests found.")
+       (do
+         (println "No tests found.")
+         test/*initial-report-counters*)
```

Comment:
I didn't want to hardcode this value in case someone else knew better (in his/her application). I figured the dynamic default in clojure.test made the most semantic sense here.